--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -145,16 +145,16 @@
 # Binary names
 #
 #=============================================================================
-set(SPEngine "openjk_sp.${Architecture}")
+set(SPEngine "openjk_sp")
 set(SPGame "jagame${Architecture}")
 set(SPRDVanillaRenderer "rdsp-vanilla_${Architecture}")
-set(MPEngine "openjk.${Architecture}")
+set(MPEngine "openjk")
 set(MPVanillaRenderer "rd-vanilla_${Architecture}")
-set(MPDed "openjkded.${Architecture}")
+set(MPDed "openjkded")
 set(MPGame "jampgame${Architecture}")
 set(MPCGame "cgame${Architecture}")
 set(MPUI "ui${Architecture}")
-set(JK2SPEngine "openjo_sp.${Architecture}")
+set(JK2SPEngine "openjo_sp")
 set(JK2SPGame "jospgame${Architecture}")
 set(JK2SPVanillaRenderer "rdjosp-vanilla_${Architecture}")
 set(AssetsPk3 "openjk-${Architecture}.pk3")
@@ -388,4 +388,4 @@
 if(BuildTests)
 	enable_testing()
 	add_subdirectory("tests")
-endif()
\ No newline at end of file
+endif()
