--- a/codeJK2/game/bg_pmove.cpp
+++ b/codeJK2/game/bg_pmove.cpp
@@ -6499,9 +6499,9 @@
 					qboolean ret =
 #endif // _DEBUG
 						gi.G2API_GetBoneAnimIndex( &gent->ghoul2[gent->playerModel], gent->lowerLumbarBone, (cg.time?cg.time:level.time), &currentFrame, &junk, &junk, &junk, &junk2, NULL );
-
+#ifdef _DEBUG
 					assert(ret); // this would be pretty bad, the below code seems to assume the call succeeds. -gil
-
+#endif // _DEBUG
 					strength = G_SaberLockStrength( gent );
 					if ( pm->ps->torsoAnim == BOTH_CCWCIRCLELOCK ||
 						pm->ps->torsoAnim == BOTH_BF2LOCK )
@@ -8991,4 +8991,4 @@
 	{//half grav
 		pm->ps->gravity *= 2;
 	}
-}
\ No newline at end of file
+}
